tests/EvmAcceptanceTests/test/Create2.js

```diff
@@ -0,0 +1,46 @@
+const {expect} = require("chai");
+const {web3} = require("hardhat");
+const web3_helper = require("../helper/Web3Helper");
+const helper = require("../helper/GeneralHelper");
+
+// Reference: https://dev.to/yongchanghe/tutorial-using-create2-to-predict-the-contract-address-before-deploying-12cb
+
+describe("Create2 instruction", function () {
+  const INITIAL_FUND = 1_000_000;
+  let contract;
```

Comment:
Recently we prefer to use `this.contract` instead of defining a new variable. can you please check it?

Reply:
sure!